guide/17-geocoding-guides/part7_utility_functions_for_geocode.ipynb

```diff
@@ -0,0 +1,717 @@
+{
```

Comment:
<p><code>suggest()</code> can be used to get search suggestions. For example, <code>suggest('Millennium Park',{'x': -87.62245002553522, 'y': 41.88425998326565})</code> will return a list of similarly named POIs within 'n' distance from the Point. However, suggest will not return actual geocode results, instead it will suggest search strings.  </p><p>The result of this operation is a resource representing a list of suggested matches for the input text. This resource provides the matching text, as well as a unique ID value that links a suggestion to a specific place or address. A <code>geocoder</code> must meet the following requirements to support the <code>suggest</code> operation: </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/830/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>